### DISCUSSION TITLE
Add Maybe to Coolify

### DISCUSSION THREAD

**Discussion by albertorizzi** [coollabsio/coolify#3775] [Feature Requests]
_**Maybe is a fully open-source OS for your personal finances.**_

https://github.com/maybe-finance/maybe/discussions/1263

**Comment by shahzeb1** [comment]
If someone adds a bounty to this, I can take it on.

**Comment by albertorizzi** [comment]
I created a PR to add Maybe finance as service: https://github.com/coollabsio/coolify/pull/4375

**Comment by albertorizzi** [comment]
Maybe is available with one click deploy.
